### PR TITLE
fix: Issues with plan

### PR DESCRIPTION
**Pull request by Arkoprabho**
# Fixes

- Docker
    - Base image
    - Utilities and versions installed
- Shell script
    - Spaces around if conditions
    - Function to set AWS profile for terragrunt
    - Comment output in plan: Fix formatting and trim init related details

### CODE REVIEW

File: src/terragrunt_taint.sh [hunk 1]
```diff
@@ -1,47 +1,48 @@
...
+        echo "${taintOutput}"
+        echo
+    fi
+
+    # Comment on the pull request if necessary.
+    if [ "$GITHUB_EVENT_NAME" == "pull_request" ] && [ "${tfComment}" == "1" ]; then
+        taintCommentWrapper="#### \`${tfBinary} taint\` ${taintCommentStatus}
+        <details><summary>Show Output</summary>
+
+        \`\`\`
```
Comment (jayeshkapadnis) [inline src/terragrunt_taint.sh:39]: IMO, we can add a language identifier to enable syntax highlighting here 
````
\`\`\`json
````

Comment (jayeshkapadnis) [reply src/terragrunt_taint.sh:39]: or `yaml`

**Review by bhatiaparas177 — APPROVED** [review]
lgtm

File: src/terragrunt_taint.sh (hunk 1 above)
Comment (Arkoprabho) [reply src/terragrunt_taint.sh:39]: The output here is text as printed by Terraform.
I doubt we can specify a specific language here, or get meaningful syntax highlighting.

An example of the output can be seen here: https://github.com/Linx-Health/pms-infra/pull/117#issuecomment-1665508769

I think HCL would come the closest, but that might mess up the syntax highlighting of the plain text items that are printed post plan output's changes

**Review by jayeshkapadnis — APPROVED** [review]

**Pull request merged**